src/struct/Core.ts

```diff
@@ -80,14 +81,22 @@ export class Core extends Client {
 				description: command.description,
 				options: command.options,
 			};
-			await makeAPIRequest(
-				`/applications/${CONFIG.CLIENT_ID}/commands`,
-				"POST",
-				body,
-			);
+			
+			// Discord limits descriptions to 100 characters
+			if (body.description.length > 100) {
+				body.description = body.substring(0, 95) + '...';
+			}
+			// Add it to our list
+			cmdList.push(body);
 			this.commands.set(command.name, command);
 			pogger.success(`Command loaded: ${command.name}`);
 		}
+		// Use one global request to update all commands.
+		await makeAPIRequest(
+			`/applications/${CONFIG.CLIENT_ID}/commands`,
+			"PUT",
+			JSON.stringify(cmdList),
```

Comment:
does Discord API supports list of commands as arrays? If it is then everything lgtm, we can merge it. 🚀🚀🚀

Reply:
well it looks Discord supports it, we can merge it! 😎